packages/postgres/src/default-pool.ts

```diff
@@ -165,13 +165,6 @@ export const callOptionalRow = defaultPool.callOptionalRow.bind(defaultPool);
  * read results in batches, which is useful for large result sets.
  */
 export const queryCursorWithClient = defaultPool.queryCursorWithClient.bind(defaultPool);
```

Comment:
Nit: do you want to remove this export and make `queryCursorWithClient` a private member of `Pool`?